app.py

```diff
@@ -51,10 +51,7 @@ def index():
     if not (app.config["CLIENT_ID"] and app.config["CLIENT_SECRET"]):
         # This check is not strictly necessary.
         # You can remove this check from your production code.
-        return """Almost there. Did you forget to set up
-<a target=_blank
-href="https://github.com/Azure-Samples/ms-identity-python-webapp/blob/master/.env.sample">
-necessary environment variables</a> for your deployment?"""
+        return render_template('config_error.html')
```

Comment:
I like the idea of rendering the error, I just wanted it to be done via a template, since HTML inside app.py is an anti-pattern.